### ISSUE TITLE
BUG - Bot does not choose higher ball if no intermediate balls available

### ISSUE THREAD

**Issue by MikeDX** [PokemonGoF/PokemonGo-Bot#889]
### Expected Behavior

Use UltraBall to catch a pokemon if normal ball < 0.35%

### Actual Behavior

Use Pokeball and fail to catch

```
[2016-07-25 20:02:36] [#] Something rustles nearby!
[2016-07-25 20:02:37] [#] A Wild Dragonite appeared! [CP 2309] [Potential 0.64]
[2016-07-25 20:02:37] [#] IV [Stamina/Attack/Defense] = [8/15/6]
[2016-07-25 20:02:40] [x] Using Pokeball (chance: 3.06%)... (5 left!)
[2016-07-25 20:02:41] [-] Attempted to capture Dragonite- failed.. trying again!
[2016-07-25 20:02:43] [x] Using Pokeball (chance: 3.06%)... (4 left!)
[2016-07-25 20:02:43] [-] Attempted to capture Dragonite- failed.. trying again!
[2016-07-25 20:02:45] [x] Using Pokeball (chance: 3.06%)... (3 left!)
[2016-07-25 20:02:46] [-] Attempted to capture Dragonite- failed.. trying again!
[2016-07-25 20:02:47] [x] Using Pokeball (chance: 3.06%)... (2 left!)
[2016-07-25 20:02:47] [-] Attempted to capture Dragonite- failed.. trying again!
[2016-07-25 20:02:48] [x] Using Pokeball (chance: 3.06%)... (1 left!)
[2016-07-25 20:02:49] [-] Attempted to capture Dragonite- failed.. trying again!
[2016-07-25 20:02:51] [x] Using Pokeball (chance: 3.06%)... (0 left!)
[2016-07-25 20:02:51] [-] Attempted to capture Dragonite- failed.. trying again!
[2016-07-25 20:02:52] [x] Out of pokeballs, switching to farming mode...
```

(ignore the fact I ran out of balls, this is just to demonstrate. When the bug originally appeared I had ultraballs in stock and eventually the pokemon ran away)

### Steps to Reproduce

Have at least one normal pokeball, find a pokemon with a < 35% catch rate and have no great balls but have some ultraballs

The logic for which ball to use is pretty bad, and doesnt take into consideration a user may not have any mid range balls to catch with.

**Comment by earthchie** [comment]
gonna fix in #908

**Comment by TheGoldenXY** [comment]
This issue has been resolved.

/closed
